completion/handlers.go

```diff
@@ -0,0 +1,77 @@
+package completion
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/coder/serpent"
+)
+
+// FileHandler returns a handler that completes file names, using the
+// given filter func, which may be nil.
+func FileHandler(filter func(info os.FileInfo) bool) serpent.CompletionHandlerFunc {
+	return func(inv *serpent.Invocation) []string {
+		return listFiles(inv.CurWord, filter)
+	}
+}
+
+// FileListHandler returns a handler that completes a list of comma-separated,
+// file names, using the given filter func, which may be nil.
```

Comment:
So this is used when an argument or flag wants a list of files? Couldn't such a list be really big? I'm having trouble understanding when we (or another user) would want it?

Reply:
This handler was inspired by the one time I used the `--tls-key-file` and `--tls-cert-file` flags on `coder server`:
`coder server --tls-cert-file="cert.pem,../cert2.pem" --tls-key-file="key1.pem,../key2.pem"`.

I don't think there's any others it'd be useful for, but it didn't take long to write regardless.

Reply:
Ok.. I get that it doesn't take long to write, but I think this API is better off without it. If it was within `coder/coder` it'd be fine, but we have higher aspirations for this library.

Reply:
That's fair, I'll scrap it